Wabbajack.App.Blazor/Pages/Install/Installing.razor.cs

```diff
@@ -26,84 +27,36 @@ public partial class Configure
+    private AbsolutePath InstallPath => StateContainer.InstallPath;
+    private AbsolutePath DownloadPath => StateContainer.DownloadPath;
+
+    public string StatusCategory { get; set; }
+
+    private string LastStatus { get; set; }
+
+    public List<string> StatusStep { get; set; } = new();
 
-    private string StatusText { get; set; } = string.Empty;
     private InstallState InstallState => StateContainer.InstallState;
 
     private const string InstallSettingsPrefix = "install-settings-";
 
     private bool _shouldRender;
     protected override bool ShouldRender() => _shouldRender;
 
-    protected override async Task OnInitializedAsync()
+    protected override void OnInitialized()
     {
-        // var Location = KnownFolders.EntryPoint.Combine("downloaded_mod_lists", machineURL).WithExtension(Ext.Wabbajack);
-        
-        await CheckValidInstallPath();
+        Install();
         _shouldRender = true;
     }
 
-    private async Task CheckValidInstallPath()
-    {
-        if (ModlistPath == AbsolutePath.Empty) return;
-        
-        var modlist = await StandardInstaller.LoadFromFile(DTOs, ModlistPath);
-        StateContainer.Modlist = modlist;
-
-        var hex = (await ModlistPath.ToString().Hash()).ToHex();
-        var prevSettings = await SettingsManager.Load<SavedInstallSettings>(InstallSettingsPrefix + hex);
-
-        if (prevSettings.ModlistLocation == ModlistPath)
-        {
-            StateContainer.ModlistPath = prevSettings.ModlistLocation;
-            InstallPath  = prevSettings.InstallLocation;
-            DownloadPath = prevSettings.DownloadLocation;
-            //ModlistMetadata = metadata ?? prevSettings.Metadata;
-        }
-
-        // see https://docs.microsoft.com/en-us/aspnet/core/blazor/images?view=aspnetcore-6.0#streaming-examples
-        var imageStream = await StandardInstaller.ModListImageStream(ModlistPath);
-        var dotnetImageStream = new DotNetStreamReference(imageStream);
-        // setImageUsingStreaming accepts the img id and the data stream
-        await JSRuntime.InvokeVoidAsync("setImageUsingStreaming", "background-image", dotnetImageStream);
-    }
-
-    private async void SelectInstallFolder()
-    {
-        try
-        {
-            var installPath = await Dialog.ShowDialogNonBlocking(true);
-            if (installPath is not null) InstallPath = (AbsolutePath)installPath;
-        }
-        catch (Exception e)
-        {
-            Logger.LogError(e, "Exception selecting install folder");
-        }
-    }
-
-    private async void SelectDownloadFolder()
-    {
-        try
-        {
-            var downloadPath = await Dialog.ShowDialogNonBlocking(true);
-            if (downloadPath is not null) DownloadPath = (AbsolutePath)downloadPath;
-        }
-        catch (Exception e)
-        {
-            Logger.LogError(e, "Exception selecting download folder");
-        }
-    }
-
-    private async Task Install()
+    private async void Install()
```

Comment:
Never use `async void`, when doing async/await always return `Task`, `Task<T>` or the `ValueTask` equivilant. Also change `void OnInitialized()` back to `async Task OnInitializedAsync()`.

Reply:
Changing `void OnInitialized()` to `async Task OnInitializedAsync()` makes the install process block UI updates.